lib/ex_webrtc/peer_connection/configuration.ex

```diff
@@ -124,9 +129,14 @@ defmodule ExWebRTC.PeerConnection.Configuration do
   * `audio_codecs` and `video_codecs` - lists of audio and video codecs to negotiate. By default these are equal to
   `default_audio_codecs/0` and `default_video_codecs/0`. To extend the list with your own codecs, do
   `audio_codecs: Configuration.default_audio_codecs() ++ my_codecs`.
-  * `header_extensions` - list of RTP header extensions to negotiate. Refer to `t:header_extension/0` for more information.
-  * `feedbacks` - list of RTCP feedbacks to negotiate. Refer to `t:rtcp_feedback/0` for more information.
   * `features` - feature flags for some of the ExWebRTC functinalities. Refer to `t:feature/0` for more information.
+  * `rtp_header_extensions` - list of RTP header extensions to negotiate. Refer to `t:rtp_header_extension/0` for more information.
+  * `rtcp_feedbacks` - list of RTCP feedbacks to negotiate. Refer to `t:rtcp_feedback/0` for more information.
+
+  Instead of manually enabling an RTP header extension or an RTCP feedback, you may want to use a `t:feature/0` instead, which will enable
```

Comment:
```suggestion
  Instead of manually enabling an RTP header extension or an RTCP feedback, you may want to use a `t:feature/0`, which will enable
```